test/connectors/CustomsDeclareExportsConnectorSpec.scala

```diff
@@ -16,23 +16,40 @@
 
 package connectors
 
-import java.time.{Instant, LocalDateTime}
+import java.time.LocalDateTime
 import java.util.UUID
 
-import base.{CustomExportsBaseSpec, MockHttpClient, TestHelper}
+import base.{MockHttpClient, TestHelper}
+import config.AppConfig
 import connectors.CustomsDeclareExportsConnector.toXml
 import forms.CancelDeclaration
 import models.declaration.notifications.Notification
 import models.declaration.submissions.RequestType.SubmissionRequest
 import models.declaration.submissions.{Action, Submission}
 import models.requests.CancellationRequested
+import org.joda.time.DateTime
+import org.scalatest.concurrent.ScalaFutures
 import play.api.http.{ContentTypes, HeaderNames}
+import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.ws.WSClient
 import play.api.mvc.Codec
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.logging.Authorization
+import unit.base.UnitSpec
 
-class CustomsDeclareExportsConnectorSpec extends CustomExportsBaseSpec {
+import scala.concurrent.ExecutionContext.global
+
+class CustomsDeclareExportsConnectorSpec extends UnitSpec with ScalaFutures {
   import CustomsDeclareExportsConnectorSpec._
 
+  val injector = GuiceApplicationBuilder().injector()
+  val appConfig = injector.instanceOf[AppConfig]
+  val hc: HeaderCarrier =
+    HeaderCarrier(
```

Comment:
Pretty sure you can just do HeaderCarrier() and there is sensible defaults

Reply:
I'll check it, if it's possible I'll remove this from all places, thanks :)

Reply:
Authorisation is necessary, DateTime not, I'll change it

Reply:
Changed